pkg/storaged/dialog.jsx

```diff
@@ -275,7 +290,71 @@ function is_visible(field, values) {
     return !field.options || field.options.visible == undefined || field.options.visible(values);
 }
 
-const Body = ({ body, fields, values, errors, onChange }) => {
+class Revealer extends React.Component {
+    constructor() {
+        super();
+        this.state = { revealed: false };
+    }
+
+    render() {
+        return (
+            <div>
+                <a onClick={event => { if (event.button == 0) this.setState({ revealed: !this.state.revealed }); }}>
+                    {this.props.summary}
+                </a> <i className={this.state.revealed ? "fa fa-angle-down" : "fa fa-angle-right"} />
+                <br />
```

Comment:
Why do you have a `<br>`? They shouldn't exist in HTML code 99% of the time they're used (unless you really want a hard break — but you probably don't here).